etc/config/compiler-explorer.defaults.properties

```diff
@@ -42,6 +42,9 @@ remoteStorageServer=https://godbolt.org
 # it is set to 5 minutes so that it will never be triggered
 # under any normal circumstances
 compilationEnvTimeoutMs=300000
+# this timeout determines when a compilation in the queue is abandoned because it has been waiting too long and the
+# client will have given up on it by now
+compilationStaleAfterMs=100000
```

Comment:
Is 100 seconds here arbitrary? Can we match it to compilationEnvTimeoutMs so the behaviour stays relatively the same?

Reply:
There's no change in behaviour with this setting: by being 100s in default it's essentially "forever". The real change is in the amazon properties.

Making it the same as compilationEnvTimeoutMs doesn't really make sense (to me) - the timeout there is mainly to catch a bug (as it says in the comment), not to actually time out the compiles (that's done with a much shorter timeout in the execution system).